src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -527,7 +527,7 @@ private function checkAssetsPackages(ContainerBuilder $container, $legacy = fals
         $package = $container->getDefinition($packages['bar']);
         $this->assertUrlPackage($container, $package, array('https://bar2.example.com'), $legacy ? null : 'SomeVersionScheme', $legacy ? '%%s?%%s' : '%%s?version=%%s');
 
-        $this->assertEquals($legacy ? 'assets.empty_version_strategy' : 'assets._version__default', (string) $container->getDefinition('assets._package_bar')->getArgument(1));
+        $this->assertEquals('assets.empty_version_strategy', (string) $container->getDefinition('assets._package_bar')->getArgument(1));
```

Comment:
I think adding `null` as the default value here was a mistake. I opened #17605 which proposes to revert that change (that would make the Asset related tests from the FrameworkBundle pass again).